README.md

```diff
@@ -111,11 +111,12 @@ Copyright 2011-2015 Daniel Dorau
 Contributions from 0x5ubt13, Adriano Pereira Junior, Akarsh Seggemu,
 Andrea Marziali, Andy Scherzinger, Andreas Schildbach, Animesh Verma,
 bashtian, bjoernb, Björn Olsson Jarl, ButterflyOfFire, cacarrara,
-Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, isi_ko404, Jasper van der Graaf,
-Joergi, koelnkalkverbot, Larissa Yasin, ligi, Luis Azcuaga, Mateus Baptista,
-Matthias Geisler, Matthias Hunstock, MichaelRocks, Nghiem Xuan Hien, NiciDieNase,
-Noemis, Omicron, Poschi, rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal,
-Teeranai.P, Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
+Caio Volpato, cketti, codingcatgirl, entropynil, ideadapt, isi_ko404, 
+Jasper van der Graaf,Joergi, koelnkalkverbot, Larissa Yasin, ligi, 
+Luis Azcuaga, Mateus Baptista, Matthias Geisler, Matthias Hunstock,
+ ichaelRocks, Nghiem Xuan Hien, NiciDieNase, Noemis, Omicron, Poschi,
+ rotrot, Sjors van Mierlo, Stefan Medack, SubOptimal, Teeranai.P,
+ Torsten Grote, Victor Herasme, Vladimir Alabov, Yanicka
```

Comment:
Almost nice - please check the preview here: https://github.com/isiko/EventFahrplan/tree/correctingContributionsNote

Please also squash your changes into one commit and force push it onto this branch.

Reply:
Sorry, I didn't See those whitespaces 😅